webhook/models.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum
+from google.cloud import ndb
+from pydantic import BaseModel, validator
+from typing import List, Optional, Dict, Union
+
+import re
+
+
+class VTType(str, Enum):
+    file = "file"
+    url = "url"
+    domain = "domain"
+    ip_address = "ip_address"
+
+
+class VTData(BaseModel):
+    attributes: Dict
+    id: str
+    links: Dict
+    type: VTType
+
+
+class VTAPI(BaseModel):
+    data: List[VTData]
+    links: Dict
+    meta: Dict
+
+
+class EmailWrapper(BaseModel):
+    api_key: str
+    email: str
+
+    @validator("api_key")
+    def api_key_validator(cls, v):
+        if not v.isalnum():
+            raise ValueError("API key must be alphanumeric!")
+        return v
+
+    @validator("email")
+    def email_validator(cls, v):
+        regex = "^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w+$"
+
+        if not re.search(regex, v):
```

Comment:
You can also use `re.match` (Read: https://docs.python.org/3/library/re.html#search-vs-match). I think `match` is more efficient.

Reply:
"Regular expressions beginning with '^' can be used with search() to restrict the match at the beginning of the string:"
But match is probably more readable, so I'll change that.